field_encrypt.services.yml

```diff
@@ -1,4 +1,8 @@
 services:
   field_encrypt.process_entities:
     class: Drupal\field_encrypt\FieldEncryptProcessEntities
-    arguments: ['@entity.query', '@entity.manager', '@encryption', '@encrypt.encryption_profile.manager']
+    arguments: ['@entity.query', '@entity.manager', '@encryption', '@encrypt.encryption_profile.manager', '@field_encrypt.encrypted_field_value_manager']
+
+  field_encrypt.encrypted_field_value_manager:
```

Comment:
Can you explain conceptually why this is a separate service from the one noted above? 

What is each one's domain?